lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -169,6 +160,30 @@ class CacheHeaders extends Audit {
     );
   }
 
+  /**
+   * @param {Map<string, string>} headers
```

Comment:
can you add a jsdoc description to this? Don't want to be redundant with the comments below, just a general category for what would disqualify a record